File: Python3/deleteAUser.py
import requests

# Method to figger out, if the given value is a integer or an integer as a string
def is_integer(n):
    try:
        float(n)
    except ValueError:
        return False
    else:
        return True

# First ask the user to enter the id of the user
user = input("Please enter the member ID: ")
if is_integer(user):
    # Add a access-token and hand over the delete request to the server
    header={'Authorization': 'Bearer <YOUR-API-KEY>'}
    req = requests.delete('https://easyverein.com/api/stable/member/' + user + '/', headers=header)
    print(req.headers)
    print(req.status_code)
else:
    print("You gave me a wrong ID")

File: PHP/deleteAUser.php
<?php

# Set the API version
$apiVersion = 'stable'

# First ask the user to enter the id of the user
$memberID = readline("Please enter the member ID: ");
if (!is_numeric($memberID)) {
    die("You gave me a wrong ID");
}

# Creating the header with the Token in it
$opts = [
    "http" => [
        "method" => "DELETE",
        "header" => "Authorization: Bearer <YOUR-API-KEY>\r\n"
    ]
];
$context = stream_context_create($opts);
# Execute the request
$content = file_get_contents('https://easyverein.com/api/' . $apiVersion . '/member/' . $memberID . '/', false, $context);
?>
